rmgpy/data/kinetics/family.py

```diff
@@ -1788,14 +1797,27 @@ def __generateReactions(self, reactants, products=None, forward=True, prod_reson
         else:
             template = self.reverseTemplate
 
+        if len(reactants) > len(template.reactants): #if the family has one template and is unimolecular split template into multiple reactants
+            try:
+                grps = template.reactants[0].item.split()
+                template_reactants = []
+                for grp in grps:
+                    entry = deepcopy(template.reactants[0])
+                    entry.item = grp
+                    template_reactants.append(entry)
+            except AttributeError:
+                template_reactants = template.reactants
```

Comment:
What's the AttributeError that you're expecting here?

Reply:
when entry has no attribute item...I believe this may be related to logic nodes.  This section gets completely changed in atg4 to make it compatiable with automatically generated trees